inst/templates/contributing-template.txt

```diff
@@ -1,134 +1,120 @@
-in order to be usable,
-our lessons must run equally well on all three.
-
-## Using GitHub
-
-If you choose to contribute via GitHub,
-you may want to look at
-[How to Contribute to an Open Source Project on GitHub][how-contribute].
-In brief, we use [GitHub flow][github-flow] to manage changes:
-
-1.  Create a new branch in your desktop copy of this repository for each significant change.
-2.  Commit the change in that branch.
-3.  Push that branch to your fork of this repository on GitHub.
-4.  Submit a pull request from that branch to the [upstream repository][repo].
-5.  If you receive feedback,
-make changes on your desktop and push to your branch on GitHub:
-the pull request will update automatically.
+## Contributing
+
+[The Carpentries][cp-site] ([Software Carpentry][swc-site], [Data
+Carpentry][dc-site], and [Library Carpentry][lc-site]) are open source
+projects, and we welcome contributions of all kinds: new lessons, fixes to
+existing material, bug reports, and reviews of proposed changes are all
+welcome.
+
+### Contributor Agreement
+
+By contributing, you agree that we may redistribute your work under [our
+license](LICENSE.md). In exchange, we will address your issues and/or assess
+your change proposal as promptly as we can, and help you become a member of our
+community. Everyone involved in [The Carpentries][cp-site] agrees to abide by
+our [code of conduct](CODE_OF_CONDUCT.md).
+
+### How to Contribute
+
+The easiest way to get started is to file an issue to tell us about a spelling
+mistake, some awkward wording, or a factual error. This is a good way to
+introduce yourself and to meet some of our community members.
+
+1. If you do not have a [GitHub][github] account, you can [send us comments by
+   email][contact]. However, we will be able to respond more quickly if you use
+   one of the other methods described below.
+
+2. If you have a [GitHub][github] account, or are willing to [create
+   one][github-join], but do not know how to use Git, you can report problems
+   or suggest improvements by [creating an issue][issues]. This allows us to
+   assign the item to someone and to respond to it in a threaded discussion.
+
+3. If you are comfortable with Git, and would like to add or change material,
+   you can submit a pull request (PR). Instructions for doing this are
+   [included below](#using-github).
+
+Note: if you want to build the website locally, please refer to [The Workbench
+documentation][template-doc].
+
+### Where to Contribute
+
+1. If you wish to change this lesson, add issues and PR here.
+2. If you wish to change the template used for workshop websites, please refer
+   to [The Workbench documentation][template-doc].
+
+
+### What to Contribute
+
+There are many ways to contribute, from writing new exercises and improving
+existing ones to updating or filling in the documentation and submitting [bug
+reports][issues] about things that don't work, aren't clear, or are missing. If
```

Comment:
```suggestion
reports][issues] about things that do not work, are not clear, or are missing. If
```

Reply:
I believe it is more accessible to expand negative contractions such as "don't", "can't", etc.

Reply:
indeed it is! Thank you!